src/app.js

```diff
@@ -1 +1,38 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+function copyFileToDestination() {
+  const [sourcePath, destinationPath] = process.argv.slice(2);
```

Comment:
Consider using destructuring assignment for better readability: const [,, sourcePath, destinationPath] = process.argv;